pkg/service/webhook/storage.go

```diff
@@ -18,18 +23,54 @@ func NewWebhookStorage(db storage.ServiceStorage) (*Storage, error) {
 	return &Storage{db: db}, nil
 }
 
-func (whs *Storage) StoreWebhook(ctx context.Context, webhook Webhook) error {
-	return nil
+func (whs *Storage) StoreWebhook(ctx context.Context, noun, verb string, webhook Webhook) error {
+	storedWebhookBytes, err := json.Marshal(webhook)
+	if err != nil {
+		return util.LoggingErrorMsg(err, "webhook marshal")
+	}
+	return whs.db.Write(ctx, webhookNamespace, getWebhookKey(noun, verb), storedWebhookBytes)
 }
 
-func (whs *Storage) GetWebhook(ctx context.Context, id string) (*Webhook, error) {
-	return nil, nil
+func (whs *Storage) GetWebhook(ctx context.Context, noun, verb string) (*Webhook, error) {
+	webhookBytes, err := whs.db.Read(ctx, webhookNamespace, getWebhookKey(noun, verb))
+	if err != nil {
+		return nil, util.LoggingErrorMsg(err, "read db")
+	}
+
+	if webhookBytes == nil || len(webhookBytes) == 0 {
+		return nil, nil
+	}
+
+	var webhook Webhook
+	if err = json.Unmarshal(webhookBytes, &webhook); err != nil {
+		return nil, util.LoggingErrorMsgf(err, "unmarshalling webhoook with key: %s", getWebhookKey(noun, verb))
+	}
+	return &webhook, nil
 }
 
 func (whs *Storage) GetWebhooks(ctx context.Context) ([]Webhook, error) {
-	return nil, nil
+	gotWebhooks, err := whs.db.ReadAll(ctx, webhookNamespace)
+	if err != nil {
+		return nil, util.LoggingErrorMsg(err, "could not get all webhooks")
+	}
+
+	var webhooks []Webhook
+	for _, webhookBytes := range gotWebhooks {
+		var webhook Webhook
+		if err = json.Unmarshal(webhookBytes, &webhook); err == nil {
+			webhooks = append(webhooks, webhook)
+		} else {
+			logrus.WithError(err).Warn("unmarshal webhook")
+		}
+	}
+
+	return webhooks, nil
+}
+
+func (whs *Storage) DeleteWebhook(ctx context.Context, noun, verb string) error {
+	return whs.db.Delete(ctx, webhookNamespace, getWebhookKey(noun, verb))
 }
 
-func (whs *Storage) DeleteWebhook(ctx context.Context, id string) error {
-	return nil
+func getWebhookKey(noun, verb string) string {
+	return noun + "-" + verb
```

Comment:
Redis keys are typically separated with `:`. 

I would suggest using the following in this case. A nice side benefit is that redis commander understands that separator easily. 
```suggestion
	return "noun:" + noun + ":verb:" + verb
```